core/App/ehsm_napi.h

```diff
@@ -50,112 +59,73 @@ static char* StringToChar(string str)
     return retChar;
 }
 
-typedef struct {
+struct RetJsonObj{
     const int CODE_SUCCESS = 200;
```

Comment:
if you want to refine the struct, please submit another PR sperately. don't mix it in this patch.

Reply:
ok